doc/release/1.17.0-notes.rst

```diff
@@ -325,41 +359,39 @@ Error messages from array comparison tests such as
 This information makes it easier to update absolute and relative error
 tolerances.
 
-Replacement of the `fftpack`-based FFT module by the `pocketfft` library
-------------------------------------------------------------------------
-Both implementations have the same ancestor (Fortran77 `FFTPACK` by Paul N.
-Swarztrauber), but `pocketfft` contains additional modifications which
-improve both accuracy and performance in some circumstances. For FFT lengths
-containing large prime factors, `pocketfft` uses Bluestein's algorithm, which
-maintains `O(N log N)` run time complexity instead of deteriorating towards
-`O(N*N)` for prime lengths. Also, accuracy for real-valued FFTs with near-prime
-lengths has improved and is on par with complex-valued FFTs.
+Replacement of the fftpack based FFT module by the pocketfft library
+--------------------------------------------------------------------
+Both implementations have the same ancestor (Fortran77 FFTPACK by Paul N.
+Swarztrauber), but pocketfft contains additional modifications which improve
+both accuracy and performance in some circumstances. For FFT lengths containing
+large prime factors, pocketfft uses Bluestein's algorithm, which maintains
+``O(N log N)`` run time complexity instead of deteriorating towards ``O(N*N)``
+for prime lengths. Also, accuracy for real valued FFTs with near prime lengths
+has improved and is on par with complex valued FFTs.
 
 Performance improvements for integer sorts
 ------------------------------------------
-
 ``sort``, ``argsort``, ``ndarray.sort`` and ``ndarray.argsort`` now use radix
 sort as the default stable sort for integers and booleans. This is faster than
-the old default, mergesort, in the vast majority of cases.
-
+the old default mergesort in the majority of cases.
 
-Further improvements to ``ctypes`` support in ``np.ctypeslib``
---------------------------------------------------------------
+Further improvements to ``ctypes`` support in `numpy.ctypeslib`
+---------------------------------------------------------------
 A new `numpy.ctypeslib.as_ctypes_type` function has been added, which can be
-used to converts a `dtype` into a best-guess `ctypes` type. Thanks to this
+used to converts a ``dtype`` into a best-guess ``ctypes`` type. Thanks to this
```

Comment:
As written before, doesn't `ctypes` become a sphinx link to the python docs? @mattip?

Ok with not linking `dtype`, that's pretty obvious to anyone who knows enough about numpy to read the release notes.

Reply:
I'm also curious about the automatic linking, when it work and when it doesn't. IIRC NumPy links require the full `numpy.`, but also seem to recall that that was fixed. Not sure how the links work for Python documentation. @mattip Thoughts?

Reply:
There's normally a `currentmodule` set, and everything in that module and higher-level modules can be found. So say you're in `linalg`, then `svd` gets resolved, `linspace` as well, but `randint` needs to be referred to as `random.randint`. For release notes no currentmodule may be set, so then you'll need the full `numpy.xxx`. For Python the builtins should be resolved, anything in a stdlib module needs full import path

Reply:
What does a full python import path look like?